Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
     "foundationdb-bench",
     "foundationdb-bindingtester",
     "foundationdb-macros",
+    "foundationdb-profiling",
```

Comment:
TODO: because it is another crate, we can create its own Github Action workflow. In that workflow, you will be able to enable 100% profiling to test things out.

Reply:
TODO: it should also be declared in the [release-plz](https://github.com/foundationdb-rs/foundationdb-rs/blob/main/release-plz.toml) config file for automatic publish on crates.io

Reply:
maybe add the release-plz in the 3rd PR. This crate is not properly usable without event parse.

Reply:
tests are actually passing.
but yes I plane to have a fuzzing step in the action when fdb_profiling is part of the PR changes